src/client.cpp

```diff
@@ -523,6 +531,20 @@ GAME_ERROR CGameLibRetro::GetRichPresenceEvaluation(char* evaluation, size_t siz
   return GAME_ERROR_NO_ERROR;
 }
 
+GAME_ERROR CGameLibRetro::ActivateAchievement(unsigned cheevo_id, const char* memaddr)
+{
+  CCheevos::Get().ActivateAchievement(cheevo_id, memaddr);
+
+  return GAME_ERROR_NO_ERROR;
```

Comment:
Is this supposed to return `GAME_ERROR_NO_ERROR` at all times? What if something happens and `CCheevos::Get().ActivateAchievement` (which returns `void`) fails? If we don't want to handle the errors at all for now, that's fine.